client/components/activity-card-list/visible-days-limit-upsell/index.tsx

```diff
@@ -64,37 +62,37 @@ const RetentionLimitUpsell: React.FC< OwnProps > = ( { cardClassName } ) => {
 		);
 
 	return (
-		<div className="retention-limit-upsell">
-			<div className="retention-limit-upsell__next-activity">{ card }</div>
-			<div className="retention-limit-upsell__call-to-action">
-				<h3 className="retention-limit-upsell__call-to-action-header">
+		<div className="visible-days-limit-upsell">
+			<div className="visible-days-limit-upsell__next-activity">{ card }</div>
+			<div className="visible-days-limit-upsell__call-to-action">
+				<h3 className="visible-days-limit-upsell__call-to-action-header">
 					{ preventWidows(
 						translate(
-							'Restore backups older than %(retentionDays)d day',
-							'Restore backups older than %(retentionDays)d days',
+							'Restore backups older than %(visibleDays)d day',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **15** times:
`translate( 'Restore backups older than %(days)d day', 'Restore backups older than %(days)d days' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/activity-log-display-rules-from-api)